docs/user_guide/assessment/error_estimation.rst

```diff
@@ -0,0 +1,360 @@
+Since this only requires one vector of values, but
+:class:`MetricFrame` requires metrics have the
+signature :code:`f(y_true, y_pred)`, we create a
+wrapper.
+We will ignore :code:`y_pred` in this case:
+
+
+.. doctest:: error_estimation_code
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> import numpy as np
+    >>> def bootstrap_sum(y_true, y_pred):
+    ...     return np.sum(y_true)
+
+Now, we generate the data itself.
+We need a column of 0s and 1s for the 'true' values
+(which we will also supply to :class:`MetricFrame`
+as the 'predicted' values - our metic will ignore them),
+and a column of labels 'A' and 'B' for the sensitive
+feature:
+
+.. doctest:: error_estimation_code
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> y_true = np.concatenate(
+    ...     (np.ones(int(n_samples * p)), np.zeros(int(n_samples * (1 - p))))
+    ... )
+    >>> s_f = np.concatenate(
+    ...     (
+    ...         np.full(n_A_correct, "A"),
+    ...         np.full(n_B_correct, "B"),
+    ...         np.full(n_A_incorrect, "A"),
+    ...         np.full(n_B_incorrect, "B"),
+    ...     )
+    ... )
+    
+    >>> # Show the sum
+    >>> bootstrap_sum(y_true, y_true)
+    500.0
+
+Activating Bootstrapping in :code:`MetricFrame`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With our data in place, we can create our :class:`MetricFrame`
+object.
+In order to use bootstrapping, we specify the
+:code:`n_bootstrap_samples` and :code:`bootstrap_random_state`
+constructor arguments.
+The first determines the number of bootstrap samples, while
+the second allows us to reproduce our results.
+
+.. doctest:: error_estimation_code
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> from fairlearn.metrics import MetricFrame, count
+    >>> metrics = {
+    ...         'bootstrap_sum' : bootstrap_sum,
+    ...         'count' : count
+    ... }
+    >>> mf = MetricFrame(
+    ...         metrics=metrics,
+    ...         y_true=y_true,
+    ...         y_pred=y_true,
+    ...         sensitive_features={ 'SF0' : s_f },
+    ...         n_bootstrap_samples=100,
+    ...         bootstrap_random_state=13489623,
+    ...     )
+    >>> # Show what we have:
+    >>> mf.overall
+    bootstrap_sum     500.0
+    count            1000.0
+    dtype: float64
+    >>> mf.by_group
+        bootstrap_sum  count
+    SF0
+    A           480.0  600.0
+    B            20.0  400.0
+
+The underlying metrics have the nominal values we expect from above.
+
+
+Accessing the Bootstrapped Results
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now on to the bootstrap results themselves.
+We access the bootstrapped values for the overall values via
+:meth:`MetricFrame.overall_quantiles`.
+This method takes a single argument, :code:`quantiles`,
+which is a list of the quantiles we want extracted, each
+in the range :math:`[0, 1]`.
+We are going to look at the bootstrapped mean and standard
+deviation, so the quantiles we want are 0.159, 0.5 and 0.841
```

Comment:
I'm probably missing something obvious, but what's so special about 0.159?

Reply:
For a gaussian, the 1-\sigma quantile (similarly for 0.841)